ner_v2/detectors/numeral/number_range/README.md

```diff
@@ -0,0 +1,246 @@
+   | :----------------------------------------------------------: | :------: | :--------: |
+   | k uper\| se upar\| se jada \| se adhik \| के ऊपर\|  से ज्यादा \|से जादा \| से अधिक |    1     |    min     |
+   |                     kam se kam\| कम से कम                     |    -1    |    min     |
+   | jyada se jyada \| lagbhag \| ज्यादा से ज्यादा \| जादा से जादा \| लगभग |    -1    |    max     |
+   | se niche \| se kam \| se sasta \| se saste k aas paas\|  k aas pas\| k lagbhag \|  से नीचे \| से कम \| से सस्ता \| से सस्ते \|   के आस पास  \| के लगभग |    1     |    max     |
+   |                           se\|-\|से                           |    0     |  min_max   |
+
+   Here, the 1st column will contain the variants of range keywords separated by pipes, which are present before or after or in between number values and defines what type of number value is present, either min or max. 
+
+   2rd column corresponds to the position of the keyword. If 1 means it will be present after number value, -1 if present before number.  
+
+   3rd column defines the range type of number i.e if it contains min value or max value or both.    
+
+   **For example** - *mujhe 2000 se jada log chahiye kal ki rally me* . In this text, range keyword `se jada` which is present in position `1` (after number) defines that number `2000` is the minimum value defined in the text, since no maximum value specified in text, hence it will be null.
+
+
+#### Guidelines to add new detectors for number range apart from builtin ones:
+
+Create a new class `number_range_detection.py`  inside language directory.
+To start with copy the code below
+
+```python
+import re
+import os
+
+from ner_v2.constant import LANGUAGE_DATA_DIRECTORY
+from ner_v2.detectors.numeral.constant import NUMBER_REPLACE_TEXT
+from ner_v2.detectors.numeral.number_range.standard_number_range_detector import BaseNumberRangeDetector
+
+
+class NumberRangeDetector(BaseNumberRangeDetector):
+    data_directory_path = os.path.join((os.path.dirname(os.path.abspath(__file__)).rstrip(os.sep)),
+                                       LANGUAGE_DATA_DIRECTORY)
+
+    def __init__(self, entity_name, language, unit_type=None):
+        super(NumberRangeDetector, self).__init__(entity_name=entity_name,
+                                                  language=language,
+                                                  data_directory_path=NumberRangeDetector.data_directory_path,
+                                                  unit_type=unit_type)
+```
+
+Note that the class name must be `NumberRangeDetector` 
+and should inherit from `ner_v2.detectors.numeral.number_range.standard_number_range_detector.BaseNumberRangeDetector`
+
+Next, we define a custom detector. For our purposes, we will add a detector to detect number range from text of patter 'between *number1* and *number2*. It will detect number1 and number2 as minimum value and maximum value respectively.
```

Comment:
`patter` -> `pattern`

`'between *number1* and *number2*`. Closing quote missing